src/js/components/SelectMultiple/SelectMultipleValue.js

```diff
@@ -0,0 +1,225 @@
+
+        return (
+          <SelectOption
+            role="option"
+            a11yTitle={
+              value.includes(iValue)
+                ? `${iLabel} selected`
+                : `${iLabel} not selected`
+            }
+            aria-setsize={value.length}
+            aria-posinset={value.indexOf(iValue) + 1}
+            aria-selected={value.includes(iValue)}
+            aria-disabled={iDisabled}
+            plain
+            hoverIndicator
+            fill="horizontal"
+            tabIndex="0"
+            onClick={(event) => {
+              if (!iDisabled) {
+                const intermediate = [...value];
+                const index = value.indexOf(iValue);
+                if (intermediate.includes(iValue)) {
+                  onSelectChange(event, {
+                    option: iValue,
+                    value: intermediate.filter((v) => v !== iValue),
+                  });
+                  if (index !== intermediate.length - 1) {
+                    setTimeout(() => {
+                      const nextFocus = document.getElementById(
+                        `selected-${intermediate[index + 1]}`,
+                      );
+                      if (nextFocus) nextFocus.focus();
+                      const result = allOptions.find(
+                        (obj, j) =>
+                          getOptionValue(
+                            j,
+                            allOptions,
+                            valueKey || labelKey,
+                          ) === intermediate[index + 1],
+                      );
+                      setShowA11yDiv(
+                        `Unselected ${iLabel}. 
+                        Focus moved to ${getOptionLabel(
+                          allOptions.indexOf(result),
+                          allOptions,
+                          labelKey || valueKey,
+                        )}`,
+                      );
+                    }, 200);
+                  } else if (intermediate.length !== 1) {
+                    setTimeout(() => {
+                      const nextFocus = document.getElementById(
+                        `selected-${intermediate[index - 1]}`,
+                      );
+                      if (nextFocus) nextFocus.focus();
+                      const result = allOptions.find(
+                        (obj, j) =>
+                          getOptionValue(
+                            j,
+                            allOptions,
+                            valueKey || labelKey,
+                          ) === intermediate[index - 1],
+                      );
+                      setShowA11yDiv(
+                        `Unselected ${iLabel}. Focus moved to 
+                          ${getOptionLabel(
+                            allOptions.indexOf(result),
+                            allOptions,
+                            labelKey || valueKey,
+                          )}`,
+                      );
+                    }, 200);
```

Comment:
add comment for how this value was chosen?

Reply:
Done